src/03-control-flow.js

```diff
@@ -163,7 +218,13 @@ function animalCounts(animal, numberOfAnimals) {}
  *  evenOrOdd(48);
  *  //> "Even"
  */
-function evenOrOdd(a) {}
+function evenOrOdd(a) {
+  if (a % 2 == 0) {
+    return "Even"
+  } if (a % 2 !== 0) {
+    return "Odd"
```

Comment:
This is fire.  This is what we want our functions to look like.  Great spacing.  Easy to read.  Add ` ; ` after the returns and you are set.